components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any reason for not including the licence ?

Reply:
Since it is an auto-generated file the license header wasn't added.